backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -101,13 +104,15 @@ nav.menu {
     display: block;
     padding: $padding-y-navbar $padding-x-navbar;
     color: $color-navbar;
-    font-weight: normal;
+    font-weight: $font-weight-bold;
   }
 
   .icon_link {
     text-indent: 2em;
 
     &:before {
+      color: $color-8;
```

Comment:
Could we not use color variables inside of our sass files, but use dedicated variables, please? So in this case add a new variable (`$navigation-icon-color`) and give it a default of `$color-8`?

I know that the whole admin sass files are build like this, but this is not something we want to continue.

Reply:
Hi @tvdeyen many thanks for your feedbacks and advices! I replaced all missed HEX colors in the scss files,and the font-weight style, also.